openpgp/packet/encrypted_key_test.go

```diff
@@ -110,7 +110,7 @@ func TestEncryptingEncryptedKey(t *testing.T) {
 		return
 	}
 
-	if ek.KeyId != keyId || ek.Algo != PubKeyAlgoRSA {
+	if ek.KeyId != keyID || ek.Algo != PubKeyAlgoRSA {
```

Comment:
Since we have to export `KeyId`, I think having `keyID` is kinda inconsistent. I'm not sure that it's worth appeasing the linter for that